ts/Series/Treegraph/TreegraphSeries.ts

```diff
@@ -198,21 +253,18 @@ class TreegraphSeries extends OrganizationSeries {
                         startOffset: '50%'
                     }
                 },
-                useHTML: false
-            },
-            link: {
-                /**
-                 * @default 'curved'
-                 *
-                 */
-                type: 'curved'
+                enabled: true,
+                linkFormatter: (): string => '',
```

Comment:
Why do we need this function? Is it not interfering with the `linkFormat`?

Reply:
It has a lesser priority than `linkFormat`. I guess, that the best is to use the function with the lowest priority so that the users could overwrite it with `pointFormat` or `linkFormat` etc.